cpp/velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -375,6 +377,8 @@ class SubstraitToVeloxPlanConverter {
 
     bool nullAllowed_ = false;
     bool isNull_ = false;
+    bool forbidsNullSet_ = false;
+    bool isNullSet_ = false;
```

Comment:
Is `forbidsNullSet_` logically equivalent to `!nullAllowed`? And `isNullSet_` equivalent to `isNull_`? If true, we can just use the existing flags. Thanks!

Reply:
Actually not. initialized_ can be set other place not just here two null related setter method.

Reply:
> Actually not. initialized_ can be set other place not just here two null related setter method.

So we can not according to `initialized_ ` variable to ensure the null behavior